leaf-ui/index.html

```diff
@@ -205,13 +205,13 @@
 		<script src="./js/backendComm.js"></script>
 
         <div class = "attribution">
-            <img src="./images/site-logo.PNG" style="width:25%;height:25%;float:left">
+            <img src="./images/site-logo.PNG" class = "image-format" style="width:25%;height:25%;float:left; z-index: 1">
             Release 1.5 <br>
             Copyright 2015-2021.<br>Grubb Lab, Smith College<br>
             & University of Toronto. <br>
             All rights reserved.<br>
             Powered by: <br>
-            <img src="./images/jointjs.png" style="width:25%;height:25%;float:left">
+            <img src="./images/jointjs.png" style="width:25%;height:25%;float:left; z-index: 1">
```

Comment:
Could add style to css class to make cleaner. Up to you, depending on if you want to use or remove `image-format`

Reply:
I think we're going to keep this. We removed image-format

Reply:
No, z-index should not be set in the .html. These are already set in style.css.